app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -40,7 +40,10 @@ def show
 
       def create
         @config_report = ConfigReport.import(params.to_unsafe_h[:config_report], detected_proxy.try(:id))
-        process_response @config_report.errors.empty?
+        errors = @config_report.errors.empty?
+        # RABL is extremely slow for huge reports and ENC script does not care about the output
+        @config_report = {}
+        process_response errors
```

Comment:
These few lines almost doubles performance, drawback: the endpoint now returns nothing instead of the processed report. Another call needed to fetch the report from DB.

We should look into why RABL is so slow as well.